src/System.Runtime.InteropServices/ref/System.Runtime.InteropServices.csproj

```diff
@@ -18,5 +18,13 @@
   <ItemGroup>
     <None Include="project.json" />
   </ItemGroup>
+  <ItemGroup>
```

Comment:
Please file a issue to remove these ProjectReference's once you have packages with the correct APIs.